plugins/scaffolder-backend/src/scaffolder/templater/index.ts

```diff
@@ -14,13 +14,16 @@
  * limitations under the License.
  */
 
+import type { Writable } from 'stream';
+
 export interface RequiredTemplateValues {
   component_id: string;
 }
 
 export interface TemplaterRunOptions {
   directory: string;
   values: RequiredTemplateValues & object;
+  logStream?: Writable;
```

Comment:
What's the thinking about the consumption pattern of this? Thinking about passing in a write stream vs returning a read stream vs reading the entire stream and returning a buffer.

Reply:
The idea is that the job creator can create a stream which then the front end can poll to get updates and more importantly the failure log from the stream.